config/cloud.cfg.tmpl

```diff
@@ -70,8 +70,8 @@ cloud_init_modules:
 {% endif %}
  - set_hostname
  - update_hostname
-{% if variant not in ["freebsd", "netbsd"] %}
  - update_etc_hosts
+{% if variant.endswith("bsd") %}
```

Comment:
I think the logic here needs inverting (to match its previous state); we should emit these lines only if we are _not_ on BSD, right?

Reply:
duh

Reply:
@ad-m It was only this line that needed updating, to match the previous meaning. We are generalising `variant not in [...]` so this should be:

```suggestion
{% if not variant.endswith("bsd") %}
```